app/ui-message/client/messageBoxNotSubscribed.js

```diff
@@ -11,17 +11,17 @@ import './messageBoxNotSubscribed.html';
 
```

Comment:
Similarly, most of the imported things are imported `RoomHistoryManager, RoomManager, t, hasAllPermission, toastr` but not used, you should remove them. These all are on line 5,6,7,8